src/keylogger/main.cpp

```diff
@@ -13,3 +16,17 @@ int main() {
     }
     return 0;
 }
+
+#else
+int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine, int nCmdShow)
```

Comment:
If we take a look we can see 90% of the code is identical, instead of copying the same code twice, lets only #ifdef the parts that change, e.g the signature of the function